content/blog/youtrack-github.md

```diff
@@ -0,0 +1,36 @@
+---
+author: "Alex Hill"
+date: 2019-04-17
+title: Integrating YouTrack and GitHub workflows
+best: false
+---
+
+Like most technical teams we use an issue tracker to manage our work. Our tool of choice for
+this is [YouTrack](https://www.jetbrains.com/youtrack/). We also make extensive use of GitHub and to synchronise our 
+workflow across the two platforms we name our git branches with YouTrack ticket ids, so every pull request 
+corresponds to a ticket. This naming convention makes it easier to track work in progress across the two platforms, 
+plus our deploy tool can use git to look at which branches have been merged in and then auto-update the 
+corresponding YouTrack tickets once the code is deployed. 
+
+An issue moves through 5 states before deployment:
```

Comment:
Could you copy our ticket workflow https://github.com/vimc/vimc-wiki/wiki/TicketWorkflow to an article and link to that directly?

Reply:
Yes but that is a YouTrack specific workflow whereas here I wanted a more abstract "issue" workflow to make the point that in principle these could be branch and/or ticket states